docs/baking.md

```diff
@@ -0,0 +1,159 @@
+<!--
+   - SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Baking with tezos-packaging on Ubuntu
+
+Tezos-packaging provides an easy way to install and set up infrastructure for
+interacting with Tezos blockchain. This article provides a step-by-step guide for
+setting up baking instance for Tezos on Ubuntu.
+
+## Prerequisites
+
+### Installing required packages
+
+In order to run baking instance, you'll need the following Tezos binaries:
+`tezos-client`, `tezos-node`, `tezos-baker-<proto>`, `tezos-endorser-<proto>`.
+
+To install them run the following commands:
+```
+# Add PPA with Tezos binaries
+sudo add-apt-repository ppa:serokell/tezos
+sudo apt-get update
+# Install packages
+sudo apt-get install tezos-client tezos-node tezos-baker-<proto> tezos-endorser-<proto>
+```
+
+Packages for `tezos-node`, `tezos-baker-<proto>` and `tezos-endorser-<proto>` provide
+systemd units for running the corresponding binaries in the background
+
```

Comment:
I think it's fairly clear what `<proto>` is, but still we should probably list the options here somehow IMO.

Especially because it seems that the names are not exactly the same as the one from our release page (although only for capitalization AFAICT).

What do you think about and adding a short sentence here pointing at launchpad.net?
E.g. for `baker`s link to:
https://launchpad.net/~serokell/+archive/ubuntu/tezos/+packages?field.name_filter=tezos-baker&field.status_filter=published